backup-loop.sh

```diff
@@ -296,6 +299,65 @@ restic() {
   call_if_function_exists "${@}"
 }
 
+rclone() {
+  _find_old_backups() {
+    command rclone lsf --format "tp" "${RCLONE_REMOTE}:${DEST_DIR}" | grep ${BACKUP_NAME} | awk \
+            -v DESTINATION="${DEST_DIR}" \
```

Comment:
...ah, given https://github.com/itzg/docker-mc-backup/pull/51/files#r774779939 it seems like you'll want this to be
```suggestion
    command rclone lsf --format "tp" "${RCLONE_REMOTE}:${RCLONE_REPOSITORY}" | grep ${BACKUP_NAME} | awk \
            -v DESTINATION="${RCLONE_REPOSITORY}" \
```

The way to read `${RCLONE_REPOSITORY:=${DEST_DIR}}` is "If RCLONE_REPOSITORY isn't already set, set it to the value of the DEST_DIR variable". And the leading colon on the line is a way to say process the line but don't execute it.